pangolin/annotate_lineages.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python
+import subprocess, pandas as pd
+from pymongo import MongoClient
+
+DB_URI = "mongodb://samples-logs-db-svc"
+SEQ_FILE = "sequences_fasta_latest.fa"
+
+
+def main():
+    download()
+    analyse()
+    records = parse_results()
+    save_to_db(records)
+
+def download():
+    subprocess.run("rm -rf /data/*", shell=True)
+    subprocess.run(f"wget "
+        f"ftp://ftp.ebi.ac.uk/pub/databases/covid19dataportal/"
+        f"viral_sequences/sequences/{SEQ_FILE}.gz"
+        f" -O /data/{SEQ_FILE}.gz"
+        , shell=True
+    )
+    subprocess.run(f"gunzip /data/{SEQ_FILE}.gz", shell=True)
```

Comment:
I would suggest to add capture_output=True, and then check the results of a command. In case of any error, it should write it somewhere in logs